dashboard/test/ui/features/step_definitions/account_steps.rb

```diff
@@ -116,6 +116,25 @@ def create_user(name, url: '/users.json', code: 201, **user_opts)
   end
 end
 
+def admin_user
```

Comment:
🎉

Reply:
Oh! Actually... is this reasonable? It adds a floating admin user to the database... I could randomize the password each time... but it is useful for having the pilot flow integrated into our UI (acceptance) tests.